src/services/tx/txEvents.ts

```diff
@@ -24,6 +25,7 @@ export enum TxEvent {
 type Id = { txId: string; groupKey?: string } | { txId?: string; groupKey: string }
 
 interface TxEvents {
+  [TxEvent.SIGNING]: { txId?: string; safeTxHash?: string }
```

Comment:
We could leverage the `safeTxHash` as a more stable `id`. Out of scope here but worth revisiting in the future.